src/main/java/com/puppycrawl/tools/checkstyle/checks/coding/MagicNumberCheck.java

```diff
@@ -567,14 +564,9 @@ public void setConstantWaiverParentToken(String... tokens) {
      * @param list list of numbers to ignore.
      */
     public void setIgnoreNumbers(double... list) {
-        if (list.length == 0) {
```

Comment:
This is pretty self-explanatory, both create an empty list and are not different at all.